client/src/services/upload/fileService.js

```diff
@@ -0,0 +1,18 @@
+import requestService from '@/services/requestService';
+
+export default {
+    async upload(file) {
+        let formData = new FormData();
+
+        formData.append('file', file);
+
+        const response = await requestService.post('/file/upload', formData);
```

Comment:
okay, I understand your intention, it makes sense. But call resource according to REST:

```
POST /files
```